client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -637,7 +637,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 			selectedDesignHasStyleVariations || ( selectedDesign.is_virtual && isXLargeScreen );
 
 		const pickDesignText =
-			selectedDesign.design_type === 'vertical' || hasMoreInfo
+			selectedDesign.design_type === 'vertical' || hasMoreInfo || ! isXLargeScreen
```

Comment:
What's the definition of extra large screen? I get 'Continue' on everything I choose - virtual and non-virtual, i'm using a window 3k pixels wide!

Reply:
"Continue" is also used when there is a sidebar (aka `hasMoreInfo`) even when the screen is extra large screen (>1080px).

To see the "Start with <DESIGN_NAME>" button, you need to select a non-virtual design without variations (to ensure that the sidebar is not displayed) on an extra large screen: 

<img width="1278" alt="Screenshot 2023-03-31 at 12 47 52" src="https://user-images.githubusercontent.com/1233880/229100396-c4c5345e-d03b-44af-89a5-dc36eb9e3851.png">

Reply:
But now that I'm thinking about it, we can probably keep the "Continue" button at all cases. It'd simplify the code and there isn't any real benefit on having a "Start with" button.

Reply:
> "Continue" is also used when there is a sidebar (aka hasMoreInfo) even when the screen is extra large (>1080px).

Doh! of course

Reply:
> we can probably keep the "Continue" button at all cases. It'd simplify the code and there isn't any real benefit on having a "Start with" button.

True, doesn't feel like a strong benefit to me either.

Reply:
> we can probably keep the "Continue" button at all cases

Done in https://github.com/Automattic/wp-calypso/pull/75168/commits/86ece69a5b46350d2cd97869dcb191ea2520ab29